README.md

```diff
@@ -579,6 +582,39 @@ async function customMiddleware(ctx, next) {
 }
 ```
 
+## Error Handling
+
+While interacting with the Okta API, the client may experience the following HTTP errors:
+
+* 429 - This is a rate limiting exception, please see [Rate Limiting at Okta] for complete details on our rate limiting strategy.
+* 503 - Service unavailable.  An unlikely situation, see [Okta Trust] for real-time service updates.
+
+In either case you will want to retry the request.  There are two ways you can handle retries, described below.
+
+### Custom Fetch Implementation
+
+If you are running into many rate limiting errors across several APIs we suggest you implement your own [Fetch interface].  The SDK will delegate all HTTP calls to your fetch interface and you must return a promise that resolves to a [Fetch response].  This allows you to write a backoff strategy for your specific needs.
+
+To provide your own fetch implementation, provide it as the `fetch` option:
+
+```javascript
+const client = new okta.Client({
+  orgUrl: 'https://dev-1234.oktapreview.com/'
+  token: 'xYzabc', // Obtained from Developer Dashboard
+  fetch: (uri, init) => {
```

Comment:
Since this is a global client implementation, is there a use-case where a dev want to update their client to use another `fetch` instance?